### ISSUE TITLE
Parse plugin accessors compositionally

### ISSUE THREAD

**Issue by hypotext** [penrose/penrose#189]
In Style, plugin accessors seem to only be parseable as standalone expressions. However, I'd like to be able to use them in larger expressions.

Repro:

On branch `mesh` with command `runpenrose mesh-domain/Figure1.sub mesh-domain/SimplicialComplex.sty mesh-domain/mesh.dsl`:

Replace this [line](https://github.com/penrose/penrose/blob/mesh/src/mesh-domain/SimplicialComplex.sty#L50) of the Style program (which parses and compiles) with an expression like `x = ddg[v.name]["x"] + 5.5` (which does not parse).

Error:

```
penrose: mesh-domain/SimplicialComplex.sty:49:8:
unexpected 'v'
expecting '}'

CallStack (from HasCallStack):
  error, called at src/Style.hs:55:17 in main:Style
CallStack (from -prof):
  Style.parsePlugins (src/Style.hs:(53,1)-(56,47))
  ShadowMain.penroseRenderer (src/ShadowMain.hs:(79,1)-(118,46))
  ShadowMain.shadowMain (src/ShadowMain.hs:(52,1)-(63,57))
```

**Comment by hypotext** [comment]
cc @wodeni

**Comment by wodeni** [comment]
Fixed in 5675b27f2c78ca327ea6c53083a7b5285278b6ab.

**Comment by hypotext** [comment]
Works so far, thanks @wodeni
